src/entries/popup/hooks/usePoll.ts

```diff
@@ -1,21 +1,18 @@
-import { useEffect, useRef } from 'react';
+import { useEffect, useLayoutEffect, useRef } from 'react';
 
 export function usePoll(callback: () => void, delay: number) {
   const cbRef = useRef<() => void>();
-  useEffect(() => {
+
+  useLayoutEffect(() => {
```

Comment:
I think using `useEffect` here would make the cb always be one render behind, probably not really a issue based on the way we use it rn but anyway